mixins/form.jsx

```diff
@@ -193,7 +193,7 @@ module.exports = {
     if (errorType === "card_error" && cardError) {
       form.error(cardError.field, cardError.message);
     } else {
-      form.error("other", this.getIntlMessage('try_again_later'));
+      form.error("other", this.getIntlMessage('try_again_later') + " [" + errorType + "]");
```

Comment:
I don't want to localize this, because I want to be able to see a screenshare and know exactly what kind of error occured even if the language is different. This is for us, not our users